systemtest/playlist.xml

```diff
@@ -671,7 +671,7 @@
 	</test>
 	
 	<test>
-		<testCaseName>SharedClassesWorkloadTest_Softmx_IncreaseDecrease</testCaseName>
+		<testCaseName>SharedClassesWorkload_Softmx_ID</testCaseName>
```

Comment:
Can this be changed to Softmx_UpDown or something that is not mistakenly read as ID = identity ?

Reply:
Good suggestion. Updated!